### PR TITLE
Add Federation

### PR DESCRIPTION
**Pull request by fcoralsasso**
This PR adds support for Federation on 2 views of the dashboard:
- Overview
- Individuals Overview

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+  CHORD_METADATA_URL,
+  FEDERATION_URL,
+} from '../constants/constants';
+
+function fetchIndividualsFederation() {
+  return fetch(FEDERATION_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: 'auth',
```
Comment (haoyuanli) [inline src/api/api.js:11]: i dont think this is needed anymore

File: src/api/api.js [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+    return {};
+  });
+}
+
+function fetchIndividualsFederationWithParams(patientParams) {
+  return fetch(FEDERATION_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: 'auth',
```
Comment (haoyuanli) [inline src/api/api.js:32]: same as above

File: src/api/api.js [hunk 3]
```diff
@@ -125,25 +170,50 @@ function searchVariant(datasetId, start, end, referenceName) {
...
+    return {};
   }
-  return fetch(`${CHORD_METADATA_URL}/api/phenopackets?found_phenotypic_feature=${symptom}&page_size=10000`, {
-    method: 'GET',
+
+  return fetch(FEDERATION_URL, {
+    method: 'POST',
     headers: {
       'Content-Type': 'application/json',
+      Authorization: 'auth',
```
Comment (haoyuanli) [inline src/api/api.js:181]: same

File: src/api/api.js (hunk 1 above)
Comment (dashaylan) [reply src/api/api.js:11]: It won't be detrimental either, but yes, the lack of an Authorization header won't cause errors now

**Review by dashaylan — APPROVED** [review]

File: src/api/api.js (hunk 2 above)
Comment (fcoralsasso) [reply src/api/api.js]: Done.

File: src/api/api.js (hunk 1 above)
Comment (fcoralsasso) [reply src/api/api.js]: Done.

File: src/api/api.js (hunk 3 above)
Comment (fcoralsasso) [reply src/api/api.js]: Done.

**Review by haoyuanli — APPROVED** [review]

**Pull request merged**